modules/KalturaSupport/components/related/related.js

```diff
@@ -386,11 +387,15 @@
 
 			if(this.getConfig('clickUrl')){
 				try {
+					if( this.getConfig( 'openInNewTab' ) === true ) {
+						window.open(this.getConfig('clickUrl'), '_blank');
+						return;
```

Comment:
@ErezBouskila , better yet, use Guard Clauses and replace the condition logic, so you can have one return at the top. More info:
http://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html